keps/sig-cli/859-kubectl-headers/README.md

```diff
@@ -279,20 +279,18 @@ can be no version skew.
 
 * **How can this feature be enabled / disabled in a live cluster?**
 
-  This feature is enabled with the KUBECTL_COMMANDS_HEADER environment
-  variable set on the client command line. Since it only affects the client
-  kubectl, it does not affect any cluster components.
+  This feature is enabled by default in kubectl, and it can not be
+  turned off (beta).
```

Comment:
Is this the usual path for CLI features? I don't see this as very likely to be risky, but I am curious as to the reasoning for NOT allowing it to be disabled via an env var?

Reply:
This is a safer path, and I will update the doc.